lib/web/scripts/node-join/install.sh

```diff
@@ -261,6 +264,10 @@ log_cleanup_message() {
         log_only "- unload and remove Teleport launchd config ${LAUNCHD_CONFIG_PATH}/${LAUNCHD_PLIST_FILE}"
         log_only "  - launchctl unload ${LAUNCHD_CONFIG_PATH}/${LAUNCHD_PLIST_FILE}"
         log_only "  - rm -f ${LAUNCHD_CONFIG_PATH}/${LAUNCHD_PLIST_FILE}"
+
+        log_only "- remove any teleport application bundles"
+        for APP_BUNDLE in ${TELEPORT_APP_BUNDLES_darwin}; do EXAMPLE_DELETE_COMMAND+="/Applications/${APP_BUNDLE} "; done
```

Comment:
nit: this should probably be quoted

```suggestion
        for APP_BUNDLE in ${TELEPORT_APP_BUNDLES_darwin}; do EXAMPLE_DELETE_COMMAND+="'/Applications/${APP_BUNDLE}' "; done
```

Otherwise it won't be quoted in the logged `rm` command, which could become a problem if `APP_BUNDLE` ever happens to contain a space, or an asterisk, or some other special undesired character.

Alternatively, to cover every case in this script, maybe the log function should implement one of [these suggestions](https://stackoverflow.com/a/27817504)?